.github/workflows/pythonapp.yml

```diff
@@ -20,45 +20,10 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
-
+    runs-on: nixos
```

Comment:
This file should be fully removed from the PR, so the github CI continues to run.